### PR TITLE
Introduce Prettier

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-server/issues/15148

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
     "node": ">= 10.13"
   },
   "scripts": {
-    "prettier": "prettier --write '/code/static/**/*.js'",
-    "prettier-ci": "prettier --list-different '/code/static/**/*.js' || (echo '\n\nThis failure means you did not run `make format` before committing\n\n' && exit 1)"
+    "prettier": "prettier --write '/code/**/*.js'",
+    "prettier-ci": "prettier --list-different '/code/**/*.js' || (echo '\n\nThis failure means you did not run `make format` before committing\n\n' && exit 1)"
```
Comment (muffinresearch) [inline package.json:10]: Any reason to not use `prettier -c ...`

Comment (willdurand) [reply package.json:10]: Probably didn't exist when we added that in addons-frontend. Thanks!

**Comment by willdurand** [comment]
So this is currently failing because we don't rely on the docker-based env in travis-ci, mmm

**Comment by willdurand** [comment]
I am going to land this now to avoid conflicts

**Pull request merged**